package/MDAnalysis/core/groups.py

```diff
@@ -1182,17 +1214,63 @@ def select_atoms(self, sel, *othersel, **selgroups):
                 therefore have atoms that were initially absent from the
                 instance :meth:`~select_atoms` was called from.
 
+            global *selection*
```

Comment:
With numpy docs, all arguments that are mentioned in the Arguments list can be written in single backtics.